app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self):

```

Comment:
I think this magic method is not needed here, you just need to write `pass` operator as you write in the previous exception or use the `docstring`. Check it in all class exceptions of this module

Reply:
I think that this is just one of the solutions. I did this on purpose in the exceptions module to remove needless from the class method. But for example, if you suddenly need to use these exceptions again - it would not be necessary to rewrite all strings.

Reply:
Maybe you're right